CHANGELOG.rst

```diff
@@ -0,0 +1,25 @@
+=================
+Changelog History
```

Comment:
Previous versions can be added retroactively. I think this could be done pretty easily in a single PR since there aren't too many PRs historically here. This will be nice to have once we have the `sphinx` documentation going.